pandas/io/parsers.py

```diff
@@ -1747,8 +1773,8 @@ def read(self, rows=None):
         columns = list(self.orig_names)
         if not len(content):  # pragma: no cover
             # DataFrame with the right metadata, even though it's length 0
-            return _get_empty_meta(self.orig_names,
-                                   self.index_col,
+            names = self._maybe_dedup_names(self.orig_names)
+            return _get_empty_meta(names, self.index_col,
```

Comment:
Yes, that's why I'm closing #9424.  There were skipped tests added in a previous PR that I am now no longer skipping.